homepage/styles/main.css

```diff
@@ -0,0 +1,135 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.navbar {
+  margin-top: 10px;
+  margin-bottom: 20px;
+  margin-left: 20px;
+  margin-right: 20px;
+  padding-bottom: 30px;
+  padding-top: 30px;
+  font-family: "Noto Sans", sans-serif;
+}
+
+.main-nav {
+  list-style-type: none;
+  display: none;
+}
+
+.main-nav li {
+  text-align: center;
+  margin: 20px auto;
+}
+
+.nav-links {
+  font-family: "Noto Sans", sans-serif;
+  text-decoration: none;
+  font-size: 16px;
+  color: grey;
+  margin-right: 20px;
+}
+
+.logo {
+  font-family: "Noto Sans JP", sans-serif;
+  display: inline-block;
+  font-size: 30px;
+  margin-top: 10px;
+  margin-left: 100px;
+  text-decoration: none;
+  color: #60ebaf;
+}
+
+.active {
+  display: block;
+}
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 0;
+    height: 70px;
+    align-items: center;
+  }
+
+  .main-nav {
+    display: flex;
+    margin-right: 30px;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  .main-nav li {
+    margin: 0;
+  }
+
+  .nav-links {
+    margin-left: 16pt;
```

Comment:
what is `pt`?

Reply:
Point size value. I just realized that em, px and % are recommended size value for CSS